src/cgroups/v2/systemd_manager.rs

```diff
@@ -172,8 +176,8 @@ impl SystemDCGroupManager {
             }
         }
 
-        common::write_cgroup_file(full_path.join(CGROUP_PROCS), &pid)?;
-        Ok(full_path)
+        common::write_cgroup_file(self.full_path.join(CGROUP_PROCS), &pid)?;
+        Ok(())
```

Comment:
I wonder if this method can just end with `common::write_cgroup_file(self.full_path.join(CGROUP_PROCS), &pid)`

Reply:
Done